pkg/operator2/starter.go

```diff
@@ -27,16 +29,42 @@ import (
 const (
 	resync = 20 * time.Minute
 
-	authConfigResource = `
+	// TODO unpause when ready
+	defaultOperatorConfig = `
 apiVersion: authentication.operator.openshift.io/v1alpha1
 kind: AuthenticationOperatorConfig
 metadata:
   name: ` + globalConfigName + `
 spec:
   managementState: Paused
 `
+
+	// TODO figure out the permanent home for top level CRDs and default CRs
+	defaultAuthentication = `
+apiVersion: config.openshift.io/v1
+kind: Authentication
+metadata:
+  name: ` + globalConfigName + `
```

Comment:
@stlaz I am doubting the usefulness of this.

Reply:
It prevents unnecessary errors if you're changing the name of that resource globally (should it ever change). Feel free to remove if you think that that should never be a problem.

Reply:
Changing it would break every operator (along with all other consumers).  I will leave it alone since it does let me trace usages (maybe that is useful).